bin/ruby-build

```diff
@@ -1040,6 +1052,15 @@ use_homebrew_yaml() {
   fi
 }
 
+use_freebsd_yaml() {
+  if is_freebsd; then
+    local libyaml_prefix="$(freebsd_package_prefix libyaml)"
+    if [ "$libyaml_prefix" ]; then
```

Comment:
Nit: although it might be functionally equivalent, I do like the explicitness of testing strings for not being blank, i.e.
```suggestion
    if [ -n "$libyaml_prefix" ]; then
```

Reply:
Explicit checks added.